cve_bin_tool/cli.py

```diff
@@ -484,16 +480,16 @@ def main(argv=None, outfile=sys.stdout):
                 LOGGER.info(f"Known CVEs in {affected_string}:")
 
                 # Creates a Object for OutputEngine
-                output = OutputEngine(modules=scanner.all_cves, filename=args.filename)
+                output = OutputEngine(modules=scanner.all_cves, filename=args.output)
```

Comment:
Hm, this is weird to read, and I don't think that's just because I've read it first the other way.  I think maybe we should go with outputfile (and maybe `--output-file` so this is more clear.)

Reply:
Sure